### PR TITLE
Add childhood vision

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-10-18-Reply-to-Childhood-Vision.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+> A clear mental picture appears to me when I try to imagine who I want to be
+
+This sounds like a great writing prompt for a "What to write about" column.
+
+What would YOU image?
+
+[Started as a comment, but was getting too long]
+
+My idea is less abstract, and more material I guess. I share the idea of this scholar aurea though.
```
Comment (MsSlacker) [inline _posts/2019-10-18-Reply-to-Childhood-Vision.md:24]: Is this "aurea" a person? Then use upper case.

File: _posts/2019-10-18-Reply-to-Childhood-Vision.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+I have already laid out my goals for the next 5 years:
+
+- [The Bigger Picture][2]
+
+The first item of which I've been trying to reach for a while, but with the current price of housing has taken a long time (and it is still going to take a while):
```
Comment (MsSlacker) [inline _posts/2019-10-18-Reply-to-Childhood-Vision.md]: "of which" sounds horrendous.

File: _posts/2019-10-18-Reply-to-Childhood-Vision.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+- A house in the countryside with lots of green.
+
+On this front I hopefully will be able to share something more about it sometime next year.
```
Comment (MsSlacker) [inline _posts/2019-10-18-Reply-to-Childhood-Vision.md]: Delete "it".

File: _posts/2019-10-18-Reply-to-Childhood-Vision.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+The other points in there are easier.
+
+Apart the happiness one.
+
+I feel that as humans we're always envy, always looking at others. It is not good nor healthy for us. You should never compare yourself to others. You are different so you're no lower or higher than other people, you are just you.
```
Comment (MsSlacker) [inline _posts/2019-10-18-Reply-to-Childhood-Vision.md]: Envy - > envious

File: _posts/2019-10-18-Reply-to-Childhood-Vision.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+Focus on what makes you happy and what you wanna do and who you wanna be.
```
Comment (MsSlacker) [inline _posts/2019-10-18-Reply-to-Childhood-Vision.md]: Too many "and".

File: _posts/2019-10-18-Reply-to-Childhood-Vision.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+Back to the house:
+
+I guess I love what it represents. The freedom of living and working surrounded by nature after living in big cities my whole life I have enough.
```
Comment (MsSlacker) [inline _posts/2019-10-18-Reply-to-Childhood-Vision.md]: "I have enough" seems disjointed from the sentence.

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-10-18-Reply-to-Childhood-Vision.md (hunk 6 above)
Comment (valeIT) [reply _posts/2019-10-18-Reply-to-Childhood-Vision.md]: Change it entirely. Not sure about the end of the sentence though

File: _posts/2019-10-18-Reply-to-Childhood-Vision.md (hunk 1 above)
Comment (valeIT) [reply _posts/2019-10-18-Reply-to-Childhood-Vision.md:24]: It is NOT. =)

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**